src/client/pages/Offer/index.tsx

```diff
@@ -144,6 +147,19 @@ export const OfferPage = ({
     [marketLabel, trackOfferEvent],
   )
 
+  useEffect(() => {
+    if (isProductSelectorEnabled) {
+      const selectedQuotes = (selectedBundleVariant?.bundle.quotes ?? []).map(
+        (quote) => quote.insuranceType as InsuranceType,
+      )
+      setSelectedInsuranceTypes(selectedQuotes)
+    }
```

Comment:
This is a little confusing since it's a list of selected insurance types, no?
Suggestion: rename variable

Question: why do we do anything if `selectedBundleVariant = undefined`?
Suggestion:  return early in that case instead of `... ?? []`

```js
if (isProductSelectorEnabled && selectedBundleVariant) {
  const insuranceTypes = selectedBundleVariant.bundle.quotes.map(
    (quote) => quote.insuranceType as InsuranceType
  );
  setSelectedInsuranceTypes(insuranceTypes);
}
```

Reply:
You're right. Pushing requested changes :)